FuncGodot Manual/pages/tips_runtime_building.html

```diff
@@ -0,0 +1,49 @@
+        <h2>Setting up Runtime map building in Godot</h2>
+        <p>Runtime map building involves using the method <code>verify_and_build()</code> available under the <a href="ref_func_godot_map.html" target="main">FuncGodotMap</a> node. The idea behind this, is that when you build a map at runtime, you set the <code>map_file</code> field, and then call this method. When this is done, this builds all the entities in the normal way, calling <code>_func_godot_apply_properties(entity_properties: Dictionary)</code> on these entities.</p>
+        <p>When the map builds, it will emit the signals <code>build_complete</code> or <code>build_failed</code>, which you can then connect to, to handle errors gracefully and to run any post-build code you need to run.</p>
+        <p>If you need the UV2 information for baking lightmaps, you can then call FuncGodotMap's <code>unwrap_uv2()</code> method. This generates the UV2 information, which emits the signal <code>unwrap_uv2_complete</code> when it is complete. It is at this point you may bake your Lightmaps.
+        <blockquote><i>As of Godot 4.2, runtime lightmap baking is not currently possible, so instead you should consider using a different lighting solution here.</i></blockquote>
+        An example of runtime map building is as follows:</p>
+        <pre><code>extends Node
+
+@onready var func_godot_map := $FuncGodotMap as FuncGodotMap
+
+# We can connect the various build signals of the FuncGodotMap 
+# in order to add more functionality after building.
+func _ready():
+    func_godot_map.connect("build_complete", _build_complete)
+    func_godot_map.connect("build_failed", _build_failed)
+    func_godot_map.connect("unwrap_uv2_complete", _unwrap_uv2_complete)
+    func_godot_map.verify_and_build()
+
+# We need to wait for the build to finish before unwrapping mesh UV2s for lightmap baking.
+# Currently lightmap baking cannot be performed at runtime, so it's not terribly useful yet.
+# Consider creating a procedural Voxel GI solid class entity that can be baked at runtime instead.
+func _build_complete() -> void:
+    print("Success! Unwrapping UV2...")
+    func_godot_map.unwrap_uv2()
+
+func _build_failed() -> void:
+    printerr("Failed to build the map file! :(")
+
+func _unwrap_uv2_complete() -> void:
+    print("UV2 unwrapping completed!")</code></pre>
+    <p>An example of this can be found in <a href="https://github.com/func-godot/func_godot_runtime_building" target="_blank">func_godot_runtime_building repository</a>.
+    <blockquote><i>NOTE: This approach has limitations. You will need to figure out solutions to anything that needs baking, or for any scene changes that need to be made after the map is built.</i></blockquote>
```

Comment:
We already cover this both in the lightmap explanation and in the code example. We can omit this whole Note entirely.

Reply:
I was on the fence about this. Ultimately I made this change, but you could make a reasonable argument that it should be mentioned as there's limitations: I.E. 3D Particle Emission points being generated for 3D isn't currently possible at runtime etc. VoxelGI baking, Navmesh baking etc. Ultimately I didn't include it because I think it's probably pretty obvious...

Interested to hear your thoughts